### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by r5us**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-KinashServer by Andriy332
-Version: 1.3
+------------
+KinashServer; by Andriy332;
```
Comment (andriy332) [inline README.md:2]: Change to 
Kinash server by @andriy332

**Comment by r5us** [comment]
ok

**Comment by r5us** [comment]
Re-Check pls

**Comment by andriy332** [comment]
Looks good

**Comment by andriy332** [comment]
Merge this pull request

**Comment by r5us** [comment]
ok

**Pull request merged**